apps/coscrad-frontend/src/components/resources/playlists/playlist-index.presenter.tsx

```diff
@@ -38,10 +39,12 @@ export const PlaylistIndexPresenter = ({ entities: playlists }: PlaylistIndexSta
         { propertyKey: 'id', headingLabel: 'Link' },
         { propertyKey: 'episodes', headingLabel: 'Number of Episodes' },
         { propertyKey: 'name', headingLabel: 'Playlist' },
+        { propertyKey: 'url', headingLabel: 'Play' },
     ];
 
     const cellRenderersDefinition: CellRenderersDefinition<IPlayListViewModel> = {
         id: renderAggregateIdCell,
+        url: renderAggregateUrlCell,
```

Comment:
As I said in the previous PR, you do not render the audio in the index view. You hook into the audio for the episodes in the playlist detail view, because one playlist has several episodes, and each one has a `mediaItemUrl`.

Reply:
fixed